documentation/book/security.adoc

```diff
@@ -32,23 +36,34 @@ The public key is used by the Kafka brokers to verify the identity of Kafka clie
 Contains all the Kafka broker private and public keys (certificates signed with the cluster CA).
 `<cluster-name>-zookeeper-nodes`::
 Contains all the Zookeeper node private and public keys (certificates signed with the cluster CA).
+`<cluster-name>-topic-operator-certs`::
+Contains the private and public keys (certificates signed with the cluster CA) used for encrypting communication between the Topic Operator and Kafka / Zookeeper.
 
 All the keys are 2048 bits in size and are valid for 365 days from initial generation.
 
 NOTE: "certificates rotation" for generating new ones on their expiration will be supported in future releases.
 
-=== Kafka Listeners
+=== Interbroker Kafka Communication
 
-The data encryption is provided on two different listeners exposed by each Kafka broker.
+Communication between brokers is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
 
-Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
+=== Internodal Zookeeper Communication
```

Comment:
Since we nowencrypt all Zoo traffic, what about calling it just _Zookeeper communication_.